events/app/controllers.js

```diff
@@ -78,10 +82,12 @@ export async function renderEventSeries(ctx, next) {
 export async function renderEventsList(ctx, next) {
   const page = ctx.request.query.page ? Number(ctx.request.query.page) : 1;
   const paginatedEvents = await getPaginatedEventPromos(page);
+  const globalAlert = ctx.intervalCache.get('globalAlert');
   const description = 'Choose from an inspiring range of free talks, tours, discussions and more, all designed to challenge how we think and feel about health.';
 
   ctx.render('pages/events', {
     pageConfig: createPageConfig({
+      globalAlert: globalAlert,
```

Comment:
In JSX land we'll be able to add this as a dep to `DefaultPageLayout`.

To make this easier in nunjucks, we could add it to the globals?
https://github.com/wellcometrust/wellcomecollection.org/blob/master/server/view/render.js#L9-L13

(but not a blocker, just a thought)

Reply:
Also might be easier when adding new pages not to forget it.

Reply:
Yep – another very good point! I'll move it to globals; having to write it in every controller feels slightly broken.